ads/google/adsense.js

```diff
@@ -17,6 +17,21 @@
 import {validateData} from '../../3p/3p';
 import {setStyles} from '../../src/style';
 import {camelCaseToDash} from '../../src/string';
+import {user} from '../../src/log';
+
+/** @const {Array<number>} */
```

Comment:
Perhaps no need for a list if there's only one value so far.

Reply:
I was wondering if we will have different approved values or not. Seemed like a generic solution would be better in this case. Maybe @charliereams has something to say on that.

Reply:
I don't think I've heard of any such plans, but Charlie can correct me if I'm wrong here.